### PR TITLE
parametrize ES USER secret and improve manifest validation

### PR DESCRIPTION
**Pull request by rccrdpccl**

### CODE REVIEW

**Comment by rccrdpccl** [comment]
The validation part might interest you @adriengentil

**Comment by adriengentil** [comment]
> The validation part might interest you @adriengentil

nice! it seems an updated version of kubeval (...that was not able to validate my CronJob object)

File: openshift/template.yaml [hunk 1]
```diff
@@ -215,13 +215,13 @@ objects:
               - name: ES_USER
                 valueFrom:
                   secretKeyRef:
-                    key: master_user_name
-                    name: elastic-master-credentials
+                    key: ${SCRAPER_ES_USER_SECRT_KEY}
```
Comment (adriengentil) [inline openshift/template.yaml:218]:
```suggestion
                    key: ${SCRAPER_ES_USER_SECRET_KEY}
```

File: openshift/template.yaml [hunk 2]
```diff
@@ -215,13 +215,13 @@ objects:
...
+                    name: ${SCRAPER_ES_USER_SECRET_NAME}
               - name: ES_PASS
                 valueFrom:
                   secretKeyRef:
-                    key: master_user_password
-                    name: elastic-master-credentials
+                    key: ${SCRAPER_ES_PASSWORD_SECRT_KEY}
```
Comment (adriengentil) [inline openshift/template.yaml:223]:
```suggestion
                    key: ${SCRAPER_ES_PASSWORD_SECRET_KEY}
```

File: openshift/template.yaml [hunk 3]
```diff
@@ -58,13 +58,13 @@ objects:
           - name: ES_USER
             valueFrom:
               secretKeyRef:
-                key: master_user_name
-                name: elastic-master-credentials
+                key: ${SCRAPER_ES_USER_SECRT_KEY}
```
Comment (adriengentil) [inline openshift/template.yaml:61]:
```suggestion
                key: ${SCRAPER_ES_USER_SECRET_KEY}
```

**Review by adriengentil — CHANGES_REQUESTED** [review]
Looks good! I just found some typos :D

**Comment by rccrdpccl** [comment]
> Looks good! I just found some typos :D

Oh no! So this does not validate exactly how I'd like it to... but it's another step
Nice catch :)

**Comment by adriengentil** [comment]
> 



> > Looks good! I just found some typos :D
> 
> Oh no! So this does not validate exactly how I'd like it to... but it's another step Nice catch :)

yeah I would have expected `oc` to complain

**Pull request merged**